sig-cloud-provider/CONTRIBUTING.md

```diff
@@ -0,0 +1,213 @@
+# Contributing
+
+Welcome to the Kubernetes SIG Cloud Provider contributing guide.  We are excited
+about the prospect of you joining our community!
+
+## Before You Begin
+
+We strongly recommend you to understand the main [Kubernetes Contributor
+Guide](http://git.k8s.io/community/contributors/guide) and adhere to the
+contribution rules (specially signing the CLA).
+
+You can also check the [Contributor Cheat
+Sheet](/contributors/guide/contributor-cheatsheet/), with common resources for
+existing developers.
+
+Read the [developer guide].
+
+Please be aware that all contributions to Kubernetes require time and
+commitment from project maintainers to direct and review work. This is done in
+additional to many other maintainer responsibilities, and direct engagement
+from maintainers is a finite resource.
+
+### Learn about our work
+
+* [The Future of Cloud Providers in Kubernetes](https://kubernetes.io/blog/2019/04/17/the-future-of-cloud-providers-in-kubernetes/)
+* [Cloud Controller Manager](https://kubernetes.io/docs/concepts/architecture/cloud-controller/)
+
+## Your first contribution
+
+### Adopt an issue
+
+Pick up an [issue] from the backlog by commenting on the issue that you would
+like to work on it.  Be sure to mention the author of the issue as well as the
+SIG Cloud Provider members.
+
+**Note:** Don't do this unless you will start work on the issue within a few
+days of being assigned.
+
+**Note:** GitHub only allows issues to be assigned to GitHub accounts that are
+part of the organization.
+
+**Picking your first issue**
+
+For your first issue, we recommend picking an issue labeled with "good first
+issue" from the [issue] backlog.  Work with active members of the SIG to find a
+suitable issue if you need help.
+
+**Picking the right size of issue**
+
+Be sure to pick up an issue that is appropriate to the time you are able to
+commit.  We recommend first time contributors start with small or medium
+issues.
+
+Following are very rough estimates, but are best effort only.  They assume you
+have a development environment already set up and are able to build a kubectl
+binary and use it against a cluster.  These estimates assume some knowledge of
+Go.
+
+- `size/S`
+  - 4-10 hours
+- `size/M`
+  - 10-20 hours
+- `size/L`
+  - 20+ hours
+- `size/XL`
+  - 40-80 hours
```

Comment:
i'm not a huge fan of the time estimates, i wonder if we could talk about complexity here to supplement?

eg
```
- `size/S`
  - simple complexity, good for novices to project (4-10 hours)
- `size/M`
  - moderate complexity (10-20 hours)
- `size/L`
  - high complexity (20+ hours)
- `size/XL`
  - very high complexity, might require help from community members (40-80 hours)
```